pymc/distributions/timeseries.py

```diff
@@ -157,15 +158,30 @@ def dist(
 
         mu = at.as_tensor_variable(floatX(mu))
         sigma = at.as_tensor_variable(floatX(sigma))
-        if steps is None:
-            raise ValueError("Must specify steps parameter")
+        shape = kwargs.get("shape", None)
+
+        # Get number of steps directly or from shape
+        if steps is not None and shape is not None:
+            steps = Assert("steps do not match last shape dimension")(
```

Comment:
Would need a test where this assert fails

Reply:
That test is already included?

Reply:
Nevermind its not

Reply:
Ok it is now